rest/route/project_vars_test.go

```diff
@@ -0,0 +1,131 @@
+package route
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/http"
+	"testing"
+
+	"github.com/evergreen-ci/evergreen/rest/data"
+	"github.com/evergreen-ci/evergreen/rest/model"
+	"github.com/evergreen-ci/gimlet"
+	"github.com/stretchr/testify/suite"
+)
+
+////////////////////////////////////////////////////////////////////////
+//
+// Tests for GET /rest/v2/projects/{project_id}/variables
+
+type GetVarsSuite struct {
```

Comment:
this is fine, but maybe the next time you write a test let's talk about subtests